### PR TITLE
PublicationSubcriber

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: src/Subscriber/PublicationSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+ *
+ * @author Conduction <info@conduction.nl> Barry Brands <barry@conduction.nl>
+ *
+ * @license EUPL <https://github.com/ConductionNL/contactcatalogus/blob/master/LICENSE.md>
+ *
+ * @category Subscriber
+ */
+class PublicationSubscriber implements EventSubscriberInterface
+{
+    const PUBLICATION_REFERENCE = 'https://openwoo.app/schemas/publication.schema.json';
```
Comment (rjzondervan) [inline src/Subscriber/PublicationSubscriber.php:26]: Shouldn't this be declared public?

**Review by rjzondervan — APPROVED** [review]

File: src/Subscriber/PublicationSubscriber.php (hunk 1 above)
Comment (bbrands02) [reply src/Subscriber/PublicationSubscriber.php:26]: It isn't used by other services but if thats the preference it can be made public

Comment (WilcoLouwerse) [reply src/Subscriber/PublicationSubscriber.php:26]: I prefer private unless there is a reason to make it public

File: src/Subscriber/PublicationSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+            if ($validationErrors !== null) {
+                return new Response(
+                    json_encode(
+                        [
+                            "message" => 'Validation errors',
+                            'data'    => $validationErrors,
+                        ]
+                    ),
+                    403
```
Comment (WilcoLouwerse) [inline src/Subscriber/PublicationSubscriber.php]: must be 400

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

**Pull request merged**